Ktisis/Interface/Components/BoneTree.cs

```diff
@@ -32,30 +49,49 @@ public class BoneTree {
 				}
 			}
 		}
+
+		private static bool BoneDoesMatch(Bone bone) => 
```

Comment:
imo this method should be renamed to `BoneMatchesSearch` or `DoesBoneMatchSearch` or similar -- with the current name it's not quite clear *what* the bone needs to match for this to return `true`.